app/renderer/ui/containers/WalletForm.tsx

```diff
@@ -305,6 +317,13 @@ class WalletFormContainer extends
     />
   )
 
+  /*
+   * Unset the caption when the user returns to seed type selection
+   */
+  private rendererSeedBackupNextStep = () => {
```

Comment:
Please rename this method to `renderSeedBackupPrevStep` as it is being used as the previous step in the render of the SeedBackup component.